.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm-3.12
+    - 5.6
+    - 7.0
+    - 7.1
+    - nightly
+    - hhvm-3.12
 
 matrix:
-  allow_failures:
```

Comment:
As I said before, it's jsut a matter of preference. Internet is full of discussions about 4 spaces vs 2 spaces, but as we use 4 spaces everywhere why not use it on YAML files too? There is nothing wrong here in my opinion